### PR TITLE
Feature: New Login/Signup UI

### PR DESCRIPTION
**Pull request by Abhiram-vv**
Features #
Updated the Login, Signup, Reset password and Confirm password UI 

### Screenshots of the changes (If any) -

[Change your Password](https://user-images.githubusercontent.com/26832094/70120351-609a9080-1692-11ea-86b6-8f630f40a112.png)
[Reset Password](https://user-images.githubusercontent.com/26832094/70120358-642e1780-1692-11ea-9604-16c704049fdc.png)
[Login](https://user-images.githubusercontent.com/26832094/70120361-698b6200-1692-11ea-85d8-0974f8d2613b.png)
[Signup](https://user-images.githubusercontent.com/26832094/70120365-6c865280-1692-11ea-9c21-1add8cf00d5c.png)

### CODE REVIEW

File: app/views/users/sessions/new.html.erb [hunk 1]
```diff
@@ -1,49 +1,106 @@
...
+    }
+  }
+</script>
+
+
+
+ <div class="container-fluid">
+   <div class="row">
+    <div class="col-lg-6 col-md-4 middle-block">
+      <%= image_tag "SVGs/login.svg", :class => "center-block"%>
```
[the remark this answers is not in the document]
Comment (Abhiram-vv) [reply app/views/users/sessions/new.html.erb]: sure, will update it

File: app/views/users/passwords/new.html.erb [hunk 2]
```diff
@@ -1,23 +1,76 @@
+<style type="text/css">
+  .custom-input,
```
Comment (tachyons) [inline app/views/users/passwords/new.html.erb]: Keep css in separate file

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -45,3 +45,73 @@
...
+   .center-block {
+    display: block;
+    margin-right: auto;
+    margin-left: auto;
+   }
+
+  .custom-submit {
+    color: #FFFFFF;
+    height: 51px;
+    font-size: 22px !important;
```
Comment (ankitkataria) [inline app/assets/stylesheets/application.css]: It's generally a good idea to not use `!important` 
Reference - https://uxengineer.com/css-specificity-avoid-important-css/

File: app/views/users/registrations/new.html.erb [hunk 4]
```diff
@@ -1,51 +1,136 @@
...
+  .custom-input:-webkit-autofill:hover, 
+  .custom-input:-webkit-autofill:focus {
+    border-bottom: 1px solid #42B983;
+    -webkit-text-fill-color: #1C1C1C;
+    -webkit-box-shadow: 0 0 0px 1000px #fff inset;
+    transition: background-color 5000s ease-in-out 0s;
+  }  
+</style>
+
+<script type="text/javascript">
```
Comment (ankitkataria) [inline app/views/users/registrations/new.html.erb]: It'd be a good idea to move UI relevant CSS to `/public/js/UX.js` or make a different file for it.

**Comment by nitin10s** [comment]
Sign up page doesn't need to have a Password Confirmation field. The hide/unhide password functionality is used to reduce this extra field only.

Change your password doesn't have navbar.

Little margins on the sides of each page and padding between form fields would make it look better.

File: app/views/users/sessions/new.html.erb [hunk 5]
```diff
@@ -1,49 +1,60 @@
...
+          <%= render "users/shared/links" %>
+        </div>
+        <% end %>
+      </div>
+    </div>
   </div>
-
 </div>
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (tachyons) [inline app/views/users/sessions/new.html.erb]: use npm for third party css

File: app/assets/stylesheets/application.css (hunk 3 above)
Comment (Abhiram-vv) [reply app/assets/stylesheets/application.css]: updated

File: app/views/users/sessions/new.html.erb (hunk 5 above)
Comment (Abhiram-vv) [reply app/views/users/sessions/new.html.erb]: updated

**Pull request merged**